docs/en/bridges.wml

```diff
@@ -210,6 +225,28 @@
     <img src="$(IMGROOT)/tb-frontpage.png" alt="Tor Browser's Front Page">
     <br><br><br><br>
 
+    <a id="Troubleshooting"></a>
+    <h2><a class="anchor" href="#Troubleshooting">Troubleshooting</a></h2>
+    <hr>
+    <br><br>
+    <h3>Sometimes the bootstrapping gets stuck at some point, for example when
+        a bridge went offline, the connection is blocked, or when the
+        Pluggable Transport is hit by a bug.
+    </h3>
+    <br/>
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-bootstrap.png"
+      alt="Tor Browser's Bootstrapping">
+    <img src="$(IMGROOT)/bridges/tb-tor-launcher-bootstrap-failed.png"
+      alt="Tor Browser's Bootstrapping">
+
+    <p>
+    Ideally it works after some tries, using a different bridge, or
+    restarting the browser. If not, remove the Tor Browser folder and extract
+    the archive again. It should work now.
+    If it still fails, we want to
+    <a href="<page about/contact>#support">hear from you</a>!
```

Comment:
Change from contact page to support.torproject.org

Reply:
done.